// Copyright (c) 2014, Cloudera, inc.
package kudu.server;

import "common/common.proto";

// The status information dumped by a server after it starts.
// This is currently only optionally exposed on the local host
// in a text file which the server writes out at startup. In the
// future, we may expose a more generic management API via RPC
// (hence the use of protobuf)
message ServerStatusPB {
  repeated HostPortPB bound_rpc_addresses = 1;
  repeated HostPortPB bound_http_addresses = 2;
}